src/core/Settings.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "gui/input/MouseConfig.h"
```

Comment:
The include looks unused, I think we can drop that here (also getting rid of the core->gui dependency).